owncloudApp/src/main/java/com/owncloud/android/interfaces/ISecurityEnforced.kt

```diff
@@ -24,6 +24,10 @@ interface ISecurityEnforced {
     fun optionLockSelected(type: LockType)
 }
 
-enum class LockType {
-    PASSCODE, PATTERN
+enum class LockType(val value: Int) {
+    PASSCODE(0), PATTERN(1);
+
+    companion object {
+        fun toLockType(value: Int): LockType = values().first { it.value == value }
+    }
```

Comment:
We could use the same approach as we did in the other PR https://github.com/owncloud/android/pull/3547 with the `LockDelayEnforced` enum

Reply:
Perfect, completely better than before.